README.md

```diff
@@ -1,22 +1,60 @@
-# Huron Examples
+# Huron examples
 
 The following is a basic example of how to set up Huron in your own project.
 
-For complete documenatation on how to use Huron, [visit the project repo](https://github.com/alleyinteractive/huron).
+For complete documentation on how to use Huron, [visit the project repo](https://github.com/alleyinteractive/huron).
 
-## Quickstart Guide
-1. Clone this repo & the huron submodule
+## Quickstart guide
 
-    `git clone --recursive https://github.com/alleyinteractive/huron-examples.git`
+1. Clone this repo
+
+    `git clone https://github.com/alleyinteractive/huron-examples.git`
 
 2. Install node modules
 
     `cd huron-examples`
     
     `npm install`
 
-3. Run huron
+3. Run Huron:
 
     `npm run huron`
 
 4. That's it! You should now be able to view the sample prototype in your browser at: http://localhost:8080/static/prototype/sample.html
+
+## Using this repo to develop for Huron
+
+The setup is slightly more complicated if you're developing new features for huron.
+
+1. Clone this repo
```

Comment:
Consider being should specific where they should clone the directory (inside the root of this project as it will already be `.gitignore`d, right?).